libethdrivers/include/ethdrivers/raw.h

```diff
@@ -156,4 +156,11 @@ struct dma_buf_cookie {
     void* pbuf;
 };
 
+struct eth_plat_config {
```

Comment:
Given this is a driver specific configuration struct this should not be in a generic header

Reply:
This change was ARM specific. There wasn't an arch_include folder, so we put it in here. What would your recommendation be?

Reply:
The current structure of this library leaves much to be desired. I do not want to make you restructure it, so for now maybe call it `arm_eth_plat_config`, and leave a comment saying it's for the imx6/zynq and should be moved somewhere else at some point?

Reply:
Okay.